File: Test/src/T2.java
import java.util.Arrays;

public class T2 {

	public static void main(String[] args) {
		String str="";
		for (String s : args) {
           	str=str+s+" ";
        }
		String[] result=str.split(" ");
		Arrays.sort(result);
		for(String i:result)
		{
			System.out.println(i);
		}
		
	}

}

File: Test/T2.kt
fun main(args: Array<String>) {
   var str=emptyArray<String>()
   	for ( s in args) 
	{
		str=str+s
	}
	str.sort()
			
	for (i in str) 
	{
    		println(i)
	}		
}


